### PR TITLE
Adding theme "Darkshell"

### PR DESCRIPTION
**Pull request by no1crate**
By me. A dark and blue theme.

### CODE REVIEW

**Review by donut28 — CHANGES_REQUESTED** [review]
All checks passed except for one: 

image

You can fix this by setting the background colour for the menu to #0A2BFF.
_This menu was the one that failed the check:_
image

**Comment by no1crate** [comment]
Done!

**Comment by kurmachu** [comment]
> All checks passed except for one:
> 
> image
> 
> You can fix this by setting the background colour for the menu to #0A2BFF. _This menu was the one that failed the check:_ image

You don't need AAA to pass for this site.

**Comment by donut28** [comment]
> You don't need AAA to pass for this site.

Oh okay.

**Review by donut28 — APPROVED** [review]
_pov: me approving of this code_

**Comment by thekifake** [comment]
I can dig it

**Review by kurmachu — CHANGES_REQUESTED** [review]
Please check accessibility, or opt-in to ignoring accessibility. 
image
image
image

The toolbar is less important and not super required, but this might be important to you.
image

**Comment by no1crate** [comment]
Done!

**Review by donut28 — APPROVED** [review]
Looks good.

File: themes/dark/darkshell.css [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+}
+.toolbarIcon.selected.theme {
+    color: #0A2BFF;
+}
+.toolbarIcon.selected.info {
+    color: #0A2BFF;
+}
+a {
+    color: #CCCCCC !important;
+}
```
Comment (kurmachu) [inline themes/dark/darkshell.css:136]: Why not set the colors in properties?

**Comment by no1crate** [comment]
> Please check accessibility, or opt-in to ignoring accessibility. image image image
> 
> The toolbar is less important and not super required, but this might be important to you. image

All done!

**Review by kurmachu — CHANGES_REQUESTED** [review]
This did not address all accessibility points. Please scan again, or opt-out of accessibility.

**Comment by no1crate** [comment]
> This did not address all accessibility points. Please scan again, or opt-out of accessibility.

I'll prefer opting-out

**Review by kurmachu — APPROVED** [review]

**Pull request merged**